web/packages/teleport/src/features.tsx

```diff
@@ -153,8 +153,19 @@ export class FeatureUnifiedResources implements TeleportFeature {
 
   category = NavigationCategory.Resources;
 
-  hasAccess() {
-    return true;
+  hasAccess(flags: FeatureFlags) {
+    // hide if dashboard tenant
+    if (cfg.isDashboard) {
+      return false;
+    }
+    // if the user has access to any of these, show the feature. Otherwise, hide.
+    return (
+      flags.nodes ||
+      flags.applications ||
+      flags.desktops ||
+      flags.kubernetes ||
+      flags.databases
+    );
```

Comment:
were there discussions around hiding this tab? (besides dashboard) i think we wanted to move towards `not hiding things` and i know for resources, it was hidden before... but what's right going forward?

because we have this feature too: https://github.com/gravitational/teleport/blob/c851bcab14b5f96a8a5216299352473c59b546fc/web/packages/teleport/src/teleportContext.tsx#L104

i'm just confused now 😆

Reply:
just double checking, i thought we `didn't` want to hide the tab? (i perhaps misunderstood our convo this morn)